### PR TITLE
MULE-15075: Migrate ObjectStore tests to MTF - Phase 1

### PR DESCRIPTION
**Pull request by alegmarra**

### CODE REVIEW

**Comment by alegmarra** [comment]
Parameterization of different object stores will be implemented as phase 2 in https://www.mulesoft.org/jira/browse/MULE-15467

File: pom.xml [hunk 1]
```diff
@@ -20,8 +20,77 @@
...
+                        </goals>
+                        <phase>integration-test</phase>
+                    </execution>
+                </executions>
+                <configuration>
+                    <targetRuntimes>
+                        <targetRuntime>MULE:4.1.1</targetRuntime>
+                        <targetRuntime>MULE:4.1.2</targetRuntime>
+                        <targetRuntime>MULE:4.1.3</targetRuntime>
+                        <targetRuntime>MULE:4.1.4-SNAPSHOT</targetRuntime>
```
Comment (estebanwasinger) [inline pom.xml]: we should start adding `4.2.0`

File: src/test/munit/clear-test-suite.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+http://www.mulesoft.org/schema/mule/os http://www.mulesoft.org/schema/mule/os/current/mule-os.xsd">
+	<munit:config name="clear-test-suite.xml" />
+	
+	<munit:before-test name="beforeClearTest" doc:id="30af555e-9965-497c-b7d8-f5920454281f" >
+		<set-variable value="keyName"  variableName="myKey"/>
+		<set-variable value="My Value" variableName="myValue"/>
+	</munit:before-test>
+	<munit:test name="clear" doc:id="652d5900-a4b5-4f56-b67c-52a160d4ad94" description="Clears the entire store">
```
Comment (estebanwasinger) [inline src/test/munit/clear-test-suite.xml]: remove `doc:id` and `doc:name` attributes from all files and elements

File: pom.xml (hunk 1 above)
Comment (alegmarra) [reply pom.xml]: will do

File: src/test/munit/clear-test-suite.xml (hunk 2 above)
Comment (alegmarra) [reply src/test/munit/clear-test-suite.xml]: nope, this is native from MUnit from studio, and all the test implemented with MTF will have it. The only way of not having them is writing the xml by hand (or like I did in some cases after copy-paste, deleting duplicates), which is not what we want to encourage.

**Pull request merged**